### PR TITLE
[on #250 of start-jsk/rtmros_gazebo][hrpsys_gazebo_tutorials] enable samplerobot and hrp2 on kinetic and melodic

### PR DESCRIPTION
**Pull request by Naoki-Hiraoka**
This PR enables hrp2 gazebo simulation on `kinetic`.
This PR in on https://github.com/start-jsk/rtmros_gazebo/pull/236.

```
roslaunch hrpsys_gazebo_tutorials gazebo_hrp2jsk_no_controllers.launch
rtmlaunch hrpsys_gazebo_tutorials hrp2jsk_hrpsys_bringup.launch 
```
hrp2jsk

```
roslaunch hrpsys_gazebo_tutorials gazebo_hrp2jsknt_no_controllers.launch
rtmlaunch hrpsys_gazebo_tutorials hrp2jsknt_hrpsys_bringup.launch 
```
hrp2jsknt

```
roslaunch hrpsys_gazebo_tutorials gazebo_hrp2jsknts_no_controllers.launch
rtmlaunch hrpsys_gazebo_tutorials hrp2jsknts_hrpsys_bringup.launch 
```
hrp2jsknts

### CODE REVIEW

**Comment by Naoki-Hiraoka** [comment]
hrp2 gazebo simulation worked on `melodic`.

```
oslaunch hrpsys_gazebo_tutorials gazebo_hrp2jsknt_no_controllers.launch
rtmlaunch hrpsys_gazebo_tutorials hrp2jsknt_hrpsys_bringup.launch 
```
The video is 10x fast because my PC (ThinkPad T440s) is not so fast and simulation is slow.
video3

**Comment by Naoki-Hiraoka** [comment]
> Does this break indigo?
> Can we keep files (yaml, launch) for indigo as renamed file (e.g., `XXX_indigo.yaml`, `XXX_indigo.launch`)?

I fixed it.

Since `hrpsys_ros_bridge_tutorials/models/xxx.yaml` are common among `indigo`, `kinetic` and `melodic`, some dynamics parameters are changed in `indigo`. This may break `indigo`, but I think It's not likely to happen. If any error caused by this PR is reported in `indigo`, I will fix it.

**Comment by Naoki-Hiraoka** [comment]
```
roslaunch hrpsys_gazebo_tutorials gazebo_samplerobot_no_controllers.launch
rtmlaunch hrpsys_gazebo_tutorials samplerobot_hrpsys_bringup.launch
```
samplerobot

File: README.md [hunk 1]
```diff
@@ -33,7 +33,8 @@ echo "source \`rospack find hrpsys_gazebo_tutorials\`/setup.sh" >> ~/.bashrc
 Open Terminal and run gazebo
 
 ```
-roslaunch hrpsys_gazebo_tutorials gazebo_hrp2jsknt_no_controllers.launch
+roslaunch hrpsys_gazebo_tutorials gazebo_hrp2jsknt_no_controllers.launch #kinetic or melodic
+roslaunch hrpsys_gazebo_tutorials gazebo_hrp2jsknt_no_controllers_indigo.launch #indigo
```
[the remark this answers is not in the document]
Comment (Naoki-Hiraoka) [reply README.md:43]: Thank you. I fixed.
https://github.com/start-jsk/rtmros_tutorials/pull/568/commits/503432f5bf65b503bbe7ea30452bfe1ded9a0aae

File: README.md [hunk 2]
```diff
@@ -16,7 +16,8 @@ If you use closed models, you have to compile `hrpsys_ros_bridge_tutorials` afte
 Open Terminal and run gazebo
 
 ```
-roslaunch hrpsys_gazebo_tutorials gazebo_samplerobot_no_controllers.launch
+roslaunch hrpsys_gazebo_tutorials gazebo_samplerobot_no_controllers.launch #kinetic or melodic
+roslaunch hrpsys_gazebo_tutorials gazebo_hrp2jsknt_no_controllers_indigo.launch #indigo
```
Comment (pazeshun) [inline README.md:20]: typo: `gazebo_hrp2jsknt_no_controllers_indigo.launch` -> `gazebo_samplerobot_no_controllers_indigo.launch`

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (pazeshun) [reply README.md:43]: The issue is moved to https://github.com/start-jsk/rtmros_tutorials/pull/568#discussion_r423586336

File: README.md (hunk 2 above)
Comment (Naoki-Hiraoka) [reply README.md]: I'm Sorry. I fixed.
https://github.com/start-jsk/rtmros_tutorials/pull/568/commits/1b51bcb0291a3764f18fc960493b8c0e77c2674d

Comment (pazeshun) [reply README.md]: I confirmed, thanks.

**Comment by pazeshun** [comment]
@k-okada Could you review this?

**Pull request merged**